spec/app_spec.rb

```diff
@@ -49,11 +49,11 @@ def app
       name = 'good'
       StubComplianceData.any_instance.stubs(:base_name).returns(name)
       StubComplianceData.any_instance.stubs(:versions).returns(
-        [
+        StubClasses::StubCollection.new([
```

Comment:
Not a blocker, but we might want to look into using `instance_double(Aws::S3::Bucket)` here.